pydbglog/dbglogmodule.cpp

```diff
@@ -23,6 +23,19 @@
  * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
  * POSSIBILITY OF SUCH DAMAGE.
  */
+
+#include "dbglog/dbglog.hpp"
+
+#ifdef _WIN32
+#include <cmath>
+#include <cstdio>
+namespace std { 
```

Comment:
Is there any other way than expanding `std`? Is this valid according to https://en.cppreference.com/w/cpp/language/extending_std?